cudax/include/cuda/experimental/__hierarchy/dimensions.cuh

```diff
@@ -11,6 +11,7 @@
 #ifndef _CUDAX__HIERARCHY_DIMENSIONS
 #define _CUDAX__HIERARCHY_DIMENSIONS
 
+#include <cuda/std/functional>
```

Comment:
Why is this needed?

Reply:
because I was a bit more picky in the mdspan implementation and it does not pull in that header anymore.

However, below we use functors from it